wandb/sdk/launch/utils.py

```diff
@@ -154,17 +150,30 @@ def parse_wandb_uri(uri: str) -> Tuple[str, str, str]:
 
 def fetch_wandb_project_run_info(uri: str, api: Api) -> Any:
     entity, project, name = parse_wandb_uri(uri)
-    try:
-        result = api.get_run_info(entity, project, name)
-    except CommError as e:
-        raise LaunchException(e)
+    result = api.get_run_info(entity, project, name)
+    # TODO: download wandb-metadata.yaml to support legacy versions?
```

Comment:
This exeception will bubble up into cli.py and be caught by `@display_error`.  We should potentially catch it here anyway if we expose these interfaces in regular python.